test/cbmc/sources/core_sntp_cbmc_state.c

```diff
@@ -90,12 +86,14 @@ SntpContext_t * allocateCoreSntpContext()
     {
         pAuthIntf->pAuthContext = malloc( sizeof( SntpAuthContext_t ) );
         pAuthIntf->generateClientAuth = GenerateClientAuthStub;
+        pAuthIntf->validateServerAuth = ValidateServerAuthStub;
     }
 
     /* It is part of the API contract to call Sntp_Init() with the SntpContext_t
      * before any other function in core_sntp_client.h. */
     if( pContext != NULL )
     {
+        pContext->numOfServers = numOfServers;
```

Comment:
Is this required? The `Sntp_Init` API would set the `numOfServer` member of the context